usmqe_tests/web/others/test_import_cluster.py

```diff
@@ -6,73 +6,33 @@
 
```

Comment:
I will not remove them. Maybe they cannot be used in CI, but they still have some meaning. Especially for manual testing they are great.